File: 0977-squares-of-a-sorted-array/0977-squares-of-a-sorted-array.cpp
class Solution {
public:
    vector<int> sortedSquares(vector<int>& nums) {
        int n = nums.size();
        vector<int>ans;
        for(int i=0;i<n;i++){
            int ans1 = nums[i]*nums[i];
            ans.push_back(ans1);

        }
        sort(ans.begin(),ans.end());
        return ans;
    }
};

File: 0977-squares-of-a-sorted-array/0977-squares-of-a-sorted-array.java
class Solution {
    public int[] sortedSquares(int[] nums) {
       int n = nums.length;
        int [] result = new int[n];
        for(int i=0; i<nums.length; i++){
           result[i]=nums[i]*nums[i];
        }
        Arrays.sort(result);
        return result;
    }
}